bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/AngleSyntax.scala

```diff
@@ -0,0 +1,39 @@
+package edu.gemini.spModel.core
+
+/**
+ * Syntax for working with angle-like types.
+ */
+object AngleSyntax {
+  implicit class ToAngleOps[A](value: A) {
+    def angle(implicit ta: ToDegrees[A]): Angle    = ta.toAngle(value)
+    def arcsecs(implicit ta: ToDegrees[A]): Double = ta.toArcsecs(value)
```

Comment:
In cases like this (pure syntax) I think it's ethical to capture the TC instance in the ctor:

```scala
  implicit class ToAngleOps[A: ToDegrees](value: A) {
    def angle: Angle = ta.toAngle(value)
    ...
```